cni/Dockerfile

```diff
@@ -12,10 +12,12 @@ FROM --platform=linux/${ARCH} mcr.microsoft.com/cbl-mariner/base/core@sha256:a49
 FROM go AS azure-vnet
 ARG OS
 ARG VERSION
+ARG CNI_AI_PATH
```

Comment:
If we have to modify source to change this (it's a package variable, I assume)... then we can probably just hardcode it. Not sure that a build arg buys us anything.

Reply:
this is mainly mirroring what is done in cns, but I can change it to just hardcode

Reply:
actually it seems like the aiMetadata is variable is used for both cns and cni. It seems odd that we would hardcode the aiMetadata to CNI's id when CNS also uses that same path, unless we want to change that too.

Reply:
I didn't mean the value, I meant the key `aiMetadata` (which I presume is the value of `CNI_AI_PATH`) that is getting set here. If we decide to use something else, we'd have to make a code change to rename that package variable anyway, no?

Reply:
Not sure if my later commits addressed your concerns-- I've set the aiMetadata equal to the cni ai id in the source code itself.

Reply:
seeing logs being logged after addressing your suggestions after a pipeline run-- could you take another look?

Reply:
i think the reason not to hardcode was to just slightly obfuscate it from the public - not that it's a _secret_ but we're not yelling it everywhere